specs/Deployment/DeploymentAPI.md

```diff
@@ -129,9 +131,57 @@ again.
     }
 ```
 
+### DeploymentInitializeOptions
+
+#### ForceDeployment
+
+When this API may install newer version of one or more of the WinAppSDK packages (i.e. an update)
+and if there are any currently running process(es) associated with these WinAppSDK package(s) (in
+other words, if any of the WinAppSDK packages to be updated are currently in use), then this API
+will fail installing that WinAppSDK package. But this update to the WinAppSDK packages can be forced
+by using DeploymentInitializeOptions object and setting ForceDeployment option before passing it to
+this API. This option when set will shut down the application(s) associated with WinAppSDK packages,
+update the WinAppSDK packages and restart the application(s).
+
+// OPENISSUE: If this option is set when updating main package, then whether the out of process com
+server from the main package such as push Notifications needs to be explicitly restarted needs to be
+understood. If restart it is needed, then the API will handle explicitly restarting it. This is
+under investigation.
+
+When the API is updating framework package and ForceDeployment option is set, then all dependent
+packages that are NOT currently in use will be immediately re-installed to refer to the updated
+framework package and all dependent packages that are currently in use will be re-installed, after
+they shut down, to refer to the updated framework package.
+
+```C#
+    if (DeploymentManager.GetStatus().Status != DeploymentStatus.Ok)
+    {
+        // Create DeploymentInitializeOptions object and set ForceDeployment option
+        // to allow for force updating of the WinAppSDK packages even if they are currently in use.
+        var deploymentInitializeOptions = new DeploymentInitializeOptions();
+        deploymentInitializeOptions.ForceDeployment = true;
+
+        // Initialize does a status check, and if the status is not OK it will attempt to get
+        // the WindowsAppRuntime into a good state by deploying packages. Unlike a simple
```

Comment:
NIT: Remove "by deploying packages". It could involve more (e.g. installing licenses). No need to go into the details here.